extensions/postgres-flyway/src/main/resources/migration/consumer/V1__m8-consumer-sample-data.sql

```diff
@@ -0,0 +1,97 @@
+INSERT INTO public.edc_transfer_process (transferprocess_id, type, state, state_count, state_time_stamp, created_at, trace_context, error_detail, resource_manifest, provisioned_resource_set, content_data_address, deprovisioned_resources, lease_id, updated_at, transferprocess_properties) VALUES ('683098a6-0ed0-4b1a-bf19-f1b0dc67f682', 'CONSUMER', 800, 1, 1687179378080, 1687179375831, '{}', NULL, '{"definitions":[]}', NULL, NULL, '[]', NULL, 1687179378080, '{}');
+
+
+--
+-- Data for Name: edc_data_request; Type: TABLE DATA; Schema: public; Owner: edc
+--
+
+INSERT INTO public.edc_data_request (datarequest_id, process_id, connector_address, protocol, connector_id, asset_id, contract_id, data_destination, managed_resources, properties, transfer_type, transfer_process_id) VALUES ('e213fc10-f61a-4aa0-80f8-7c581651f543', '3499e4f0-1f8c-4ae6-8ca3-82812586b80f', 'http://edc:11003/api/v1/ids/data', 'ids-multipart', 'consumer', 'urn:artifact:test:1.0', 'test-contract-definition:2160b70f-d65e-4861-af88-f29b16f3fa7b', '{"properties":{"baseUrl":"https://webhook.site/e542f69e-ff0a-4771-af18-0900a399137a","method":"POST","type":"HttpData"}}', false, '{}', '{"contentType":"application/octet-stream","isFinite":true}', '3499e4f0-1f8c-4ae6-8ca3-82812586b80f');
+INSERT INTO public.edc_data_request (datarequest_id, process_id, connector_address, protocol, connector_id, asset_id, contract_id, data_destination, managed_resources, properties, transfer_type, transfer_process_id) VALUES ('34b587d1-9a1a-4bc2-a499-24a0e6975b97', '3c7606fd-e3e3-453a-bc38-e2fd6293acc8', 'http://edc:11003/api/v1/ids/data', 'ids-multipart', 'consumer', 'urn:artifact:test:1.0', 'test-contract-definition:2160b70f-d65e-4861-af88-f29b16f3fa7b', '{"properties":{"baseUrl":"https://webhook.site/e542f69e-ff0a-4771-af18-0900a399137a","method":"POST","type":"HttpData"}}', false, '{}', '{"contentType":"application/octet-stream","isFinite":true}', '3c7606fd-e3e3-453a-bc38-e2fd6293acc8');
+INSERT INTO public.edc_data_request (datarequest_id, process_id, connector_address, protocol, connector_id, asset_id, contract_id, data_destination, managed_resources, properties, transfer_type, transfer_process_id) VALUES ('5e4cb5c8-fc8e-4231-ba86-bc5dbf7b5015', '683098a6-0ed0-4b1a-bf19-f1b0dc67f682', 'http://edc:11003/api/v1/ids/data', 'ids-multipart', 'consumer', 'urn:artifact:test:1.0', 'test-contract-definition:2160b70f-d65e-4861-af88-f29b16f3fa7b', '{"properties":{"baseUrl":"https://webhook.site/e542f69e-ff0a-4771-af18-0900a399137a","method":"POST","type":"HttpData"}}', false, '{}', '{"contentType":"application/octet-stream","isFinite":true}', '683098a6-0ed0-4b1a-bf19-f1b0dc67f682');
```

Comment:
data should contain both a providing and a consuming data offer (both with transfers)